lib/Github/Api/GraphQL.php

```diff
@@ -21,9 +21,9 @@ class GraphQL extends AbstractApi
      *
```

Comment:
```suggestion
     * @param string $acceptHeaderValue
     *
```

Reply:
I've suggested to add the string typehint instead below